### PR TITLE
Create shim script for LibreOffice's `soffice`

### PR DESCRIPTION
**Pull request by wfdd**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

---

`soffice` depends on other files within the LibreOffice app.
Document conversion with the symlinked `soffice` would fail
with the following error:

```
... No Info.plist file in application bundle or no NSPrincipalClass in the Info.plist file, exiting
```

See also https://superuser.com/questions/1078968 and
https://gist.github.com/psjinx/3ad78df6290b5ba931c1.

### CODE REVIEW

File: Casks/libreoffice.rb [hunk 1]
```diff
@@ -30,6 +32,14 @@
   binary "#{appdir}/LibreOffice.app/Contents/MacOS/uri-encode"
   binary "#{appdir}/LibreOffice.app/Contents/MacOS/xpdfimport"
 
+  preflight do
+    File.open(shimscript, 'w') do |f|
+      f.puts '#!/usr/bin/env bash'
+      f.puts "#{Hbc.appdir}/LibreOffice.app/Contents/MacOS/soffice \"$@\""
```
Comment (vitorgalvao) [inline Casks/libreoffice.rb]: As of today, we can use `appdir` instead of `Hbc.appdir`.

File: Casks/libreoffice.rb [hunk 2]
```diff
@@ -21,7 +21,9 @@
   binary "#{appdir}/LibreOffice.app/Contents/MacOS/regmerge"
   binary "#{appdir}/LibreOffice.app/Contents/MacOS/regview"
   binary "#{appdir}/LibreOffice.app/Contents/MacOS/senddoc"
-  binary "#{appdir}/LibreOffice.app/Contents/MacOS/soffice"
+  # shim script (https://github.com/caskroom/homebrew-cask/issues/18809)
```
Comment (vitorgalvao) [inline Casks/libreoffice.rb]: Could you move this to the bottom? Just so it’s nearer the actual `preflight` and is easier to spot.

File: Casks/libreoffice.rb (hunk 1 above)
Comment (wfdd) [inline Casks/libreoffice.rb]: Thanks, fixed below.

File: Casks/libreoffice.rb (hunk 2 above)
Comment (wfdd) [inline Casks/libreoffice.rb]: Done!

**Comment by vitorgalvao** [comment]
Thank you. Merging when Travis passes.

**Pull request merged**